clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/health/AmazonHealthIndicator.groovy

```diff
@@ -80,7 +79,7 @@ class AmazonHealthIndicator implements HealthIndicator {
   @Scheduled(fixedDelay = 120000L)
   void checkHealth() {
     try {
-      Set<NetflixAmazonCredentials> amazonCredentials = accountCredentialsProvider.all.findAll {
+      Set<NetflixAmazonCredentials> amazonCredentials = credentialsRepository.getAll().findAll {
```

Comment:
`credentialsRepository.getAll().forEach(....)` the test for `instanceof` is always true